notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Can rephrase to use active verbs, and also add detail about the specific tools being used, e.g.</p><ul><li>Extract a stellar rotation rate from a TESS light curve using lightkurve [presumably; haven't gotten far enough yet]</li><li>Apply the Nyquist-Shannon Samping Theorem when finding stellar rotation rates by [something; gotta see]</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>